camelot/admin/action/list_action.py

```diff
@@ -399,17 +410,23 @@ class DeleteSelection( EditAction ):
     verbose_name = _('Delete')
 
     def gui_run( self, gui_context ):
-        #
-        # if there is an open editor on a row that will be deleted, there
-        # might be an assertion failure in QT, or the data of the editor 
-        # might be pushed to the row that replaces the deleted one
-        #
-        gui_context.item_view.close_editor()
-        super( DeleteSelection, self ).gui_run( gui_context )
-        # this refresh call could be avoided if the removal of an object
-        # in the collection through the DeleteObject action step handled this
-        gui_context.item_view.model().refresh()
-        gui_context.item_view.clearSelection()
+        if gui_context.item_view is not None:
+            #
+            # if there is an open editor on a row that will be deleted, there
+            # might be an assertion failure in QT, or the data of the editor
+            # might be pushed to the row that replaces the deleted one
+            #
+            gui_context.item_view.close_editor()
```

Comment:
ik DENK dat deze close_editor nt meer nodig is (tb checked in een one2many editor), volgens mij was dit een specifieke bug in Qt4, die gefixed in in Qt5 (of een memory failure langs mijn kant)

Reply:
Deze is inderdaad niet meer nodig (d.i. ik kan een rij verwijderen in een one2many editor met een open editor)